src/driver/codemirror/linkFolder/index.ts

```diff
@@ -21,6 +21,12 @@ const ENHANCEMENT_PLANTUML_SPAN_MARKER_LINE_CLASS = 'enhancement-plantuml-block-
 const ENHANCEMENT_HORIZONTAL_LINE_CLASS_LINE_CLASS = 'enhancement-horizontal-line-marker-line';
 
 export async function linkFolderOptionFunc(_context, cm, val, old) {
+    // FIXME(for:cm6): Disabled because the following logic is broken in
+    // the CM6 editor.
+    if (cm.cm6) {
+        return;
+    }
```

Comment:
For now, I've disabled several parts of the plugin that fail to load in CodeMirror 6.